stages/org.osbuild.coreos.live-artifacts.mono.meta.json

```diff
@@ -0,0 +1,68 @@
+{
+  "summary": "Build CoreOS Live ISO and PXE kernel,initramfs,rootfs",
+  "description": [
+    "This stage builds the CoreOS Live ISO and PXE kernel,initramfs,rootfs",
```

Comment:
Minor: hmm, is there a reason for no spacing in the commas?

Reply:
just that they are all PXE related.. would `kernel/initramfs/rootfs` be better?